### PR TITLE
Update device-discovery-faq.md

### PR DESCRIPTION
**Pull request by roshahno**

### CODE REVIEW

File: microsoft-365/security/defender-endpoint/device-discovery-faq.md [hunk 1]
```diff
@@ -48,23 +48,23 @@ In this mode endpoints onboarded to Microsoft Defender for Endpoint can actively
...
+You can customize the list of devices that are used to perform Standard discovery. You can either enable Standard discovery on all the onboarded devices that also support this capability (currently Windows 10+ and Windows Server 2019+ devices only) or select a subset or subsets of your devices by specifying their device tags. In this case, all other devices will be configured to run Basic discovery only. The configuration is available in the device discovery settings page.
 
 ## Can I exclude unmanaged devices from the device inventory list?
 
 Yes, you can apply filters to exclude unmanaged devices from the device inventory list. You can also use the onboarding status column on API queries to filter out unmanaged devices.
 
 ## Which onboarded devices can perform discovery?
 
-Onboarded devices running on Windows 10 version 1809 or later, or Windows 11 can perform discovery. Servers cannot perform discovery at this point.
+Onboarded devices running on Windows 10 version 1809 or later, Windows 11, Windows Server 2019 or Windows Server 2022 can perform discovery.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/device-discovery-faq.md]:
```suggestion
Onboarded devices running on Windows 10 version 1809 or later, Windows 11, Windows Server 2019, or Windows Server 2022 can perform discovery.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @roshahno, kindly commit the minor suggested changes. Many thanks.

File: microsoft-365/security/defender-endpoint/device-discovery-faq.md [hunk 2]
```diff
@@ -48,23 +48,23 @@ In this mode endpoints onboarded to Microsoft Defender for Endpoint can actively
...
 
 ## What happens if my onboarded devices is connected to my home network, or to public access point?
 
 The discovery engine distinguishes between network events that are received in the corporate network versus outside of the corporate network. By correlating network identifiers across all tenant's clients, events are differentiated between ones that were received from private networks and corporate networks. For example, if the majority of the devices in the organization report that they are connected to the same network name, with the same default gateway and DHCP server address, it can be assumed that this network is likely a corporate network. Private network devices will not be listed in the inventory and will not be actively probed.
 
 ## What protocols are you capturing and analyzing?
 
-By default, all onboarded devices running on Windows 10 version 1809 or later, or Windows 11 are capturing and analyzing the following protocols:
+By default, all onboarded devices running on Windows 10 version 1809 or later, Windows 11, Windows Server 2019 or Windows Server 2022 are capturing and analyzing the following protocols:
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/device-discovery-faq.md]:
```suggestion
By default, all onboarded devices running on Windows 10 version 1809 or later, Windows 11, Windows Server 2019, or Windows Server 2022 are capturing and analyzing the following protocols:
```

**Comment by yogkumgit** [comment]
@roshahno Please implement the suggested changes. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR. Thanks

**Pull request merged**